recipes/default.rb

```diff
@@ -5,10 +5,10 @@
   action :purge
 end
 
-execute "Setup ChefDK as default ruby" do
+execute 'Setup ChefDK as default ruby' do
   command "echo 'eval \"$(chef shell-init bash)\"' >> /root/.bashrc"
-  user "root"
-  environment( { "HOME" => "/root" } )
+  user 'root'
+  environment('HOME' => '/root')
```

Comment:
I'm not sure that it was the right thing to remove the braces here so I'll add them back in a separate commit.